### PR TITLE
Removed deprecations

### PR DESCRIPTION
**Pull request by core23**
I am targetting this branch, because this is a bugfix.

## Changelog

```markdown
### Deprecated
 - Removed deprecation for security.context
 - Removed block service deprecation
```

## Subject

Removed symfony and internal deprecation.

### CODE REVIEW

File: Admin/AdminExtension.php [hunk 1]
```diff
@@ -25,15 +25,15 @@ class AdminExtension extends AbstractAdminExtension
...
     protected $securityContext;
 
     /**
-     * @param ActionManagerInterface   $actionManager
-     * @param SecurityContextInterface $securityContext
+     * @param ActionManagerInterface $actionManager
+     * @param TokenStorageInterface  $securityContext
      */
-    public function __construct(ActionManagerInterface $actionManager, SecurityContextInterface $securityContext)
+    public function __construct(ActionManagerInterface $actionManager, TokenStorageInterface $securityContext)
```
Comment (jordisala1991) [inline Admin/AdminExtension.php]: I think you should not type hint this.

File: Admin/AdminExtension.php [hunk 2]
```diff
@@ -25,15 +25,15 @@ class AdminExtension extends AbstractAdminExtension
...
-     * @var SecurityContextInterface
+     * @var TokenStorageInterface
      */
     protected $securityContext;
 
     /**
-     * @param ActionManagerInterface   $actionManager
-     * @param SecurityContextInterface $securityContext
+     * @param ActionManagerInterface $actionManager
+     * @param TokenStorageInterface  $securityContext
```
Comment (jordisala1991) [inline Admin/AdminExtension.php]: This is `SecurityContextInterface|TokenStorageInterface`

**Comment by core23** [comment]
Can you review again @jordisala1991 ?

**Comment by jordisala1991** [comment]
Thanks @core23 !

**Comment by Soullivaneuh** [comment]
@core23 please look at the changelog update. The `Deprecated` section of the changelog is for deprecation **addition**, not fixes.

**Pull request merged**